### DISCUSSION TITLE
Trying to link Google Cloud Platform Vertex AI Instance to GitHub repository

### DISCUSSION THREAD

**Discussion by MemeRunner** [community/community#77229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to connect my Google Vertex AI notebook/instance to Github, using these instructions.

Instructions (for managed notebook): 
https://cloud.google.com/vertex-ai/docs/workbench/managed/save-to-github 

However, when I go to authenticate via the popup window that appears, I'm getting the following error. 

Error message:
remote: Support for password authentication was removed on August 13, 2021. remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication. fatal: Authentication failed for 'https://github.com/MemeRunner/aplusview.git/'

This happens after after 'Cloning a Repository' while trying to authenticate.

I'm not familiar with GitHub authentication, but anyone who has any suggestion on where to go next, I'd greatly appreciate it.

**Comment by AarishShah22** [comment]
I think you have to generate and use a personal access token, not your github password.

**Reply by MemeRunner** [reply]
Thanks @AarishShah22. I'm now using Colab Enterprise, and I don't think it works with that, though.

**Reply by queenofcorgis** [reply]
Hi @MemeRunner were you able to get this solved? If so, we'd love to know the solution - was it Colab Enterprise?

**Reply by MemeRunner** [reply]
No, I never was able to get this functioning. It's an issue with the Enterprise version of Colab
